### ISSUE TITLE
Add notdef and nbspace

### ISSUE THREAD

**Issue by RosaWagner** [googlefonts/gftools#667]
The builder should be able to add these missing glyphs if absent. And also add a square or something in the .notdef outlines if it's empty. 

cc @yanone @m4rc1e @vv-monsalve @emmamarichal

**Comment by anthrotype** [comment]
> add a square or something in the .notdef outlines if it's empty.

this is done in ufo2ft already, if I'm not mistaken

**Comment by RosaWagner** [comment]
AAAH then it would juste be an argument to add to the builder, even better.

**Comment by anthrotype** [comment]
no. it's not optional feature. ufo2ft automatically always adds a .notdef glyph if there isn't one already, and draws a box that fits the font metrics. Is it not what you are getting?

**Comment by RosaWagner** [comment]
We got to many WARNS from fontbakery about empty notdef. Does ufo2ft add the box if the glyphs is present but empty in the sources?

**Comment by RosaWagner** [comment]
Okay, so that's my issue, GF requires a .notdef with outlines. Most designer add the .notdef in their sources cause it's part of our glyphsets, but don't draw any outlines.

**Comment by anthrotype** [comment]
I see. I don't feel like changing ufo2ft to draw a notdef when present but empty, the requirement for it to be drawn is not so universally clear-cut, I think there are valid use-cases for keeping it blank
